lib/FrontendTool/FrontendTool.cpp

```diff
@@ -121,7 +121,7 @@ static bool emitMakeDependencies(DiagnosticEngine &diags,
   llvm::SmallString<256> pathBuf;
   auto escape = [&](StringRef raw) -> StringRef {
     pathBuf.clear();
-
+    
```

Comment:
Nitpick: stray indentation.

Reply:
Fixed, but git-clang-format does this, sigh.